USAGE.md

```diff
@@ -17,5 +17,12 @@ export OFX_PLUGIN_PATH=/path/to/your/tuttle/OFX
 Note: some host provide a way to set this path in their GUI. Checkout the "preference" menu.
 
 
+## Override host name
+Some OFX plugins define a specific behavior depending on the host (shame on them!).
```

Comment:
Some OFX plugins define a specific behavior depending on the host **name** (shame on them!).